packages/desktop-client/src/components/budget/envelope/EnvelopeBudgetComponents.tsx

```diff
@@ -301,6 +302,10 @@ export const ExpenseCategoryMonth = memo(function ExpenseCategoryMonth({
               borderColor: theme.tableBorder,
             }}
           >
+            <NotesButton
+              id={`${month}-${category.id}`}
```

Comment:
Can we use this id to align with the naming of the budget month notes?
```suggestion
              id={`budget-${month}-${category.id}`}
```